include/openmc/distribution_multi.h

```diff
@@ -43,6 +43,11 @@ class PolarAzimuthal : public UnitSphereDistribution {
   //! \param seed Pseudorandom number seed pointer
   //! \return Direction sampled
   Direction sample(uint64_t* seed) const;
+
+  // Observing pointers
+  auto mu() const { return mu_.get(); }
+  auto phi() const { return phi_.get(); }
```

Comment:
I personally like to see the return type here
```suggestion
  Distribution* mu() const { return mu_.get(); }
  Distribution* phi() const { return phi_.get(); }
```